syspurpose/src/syspurpose/files.py

```diff
@@ -44,8 +45,8 @@ def read_file(self):
         :return: False if the contents of the file were empty, or the file doesn't exist; otherwise, nothing.
         """
         try:
-            with open(self.path, 'r') as f:
-                self.contents = json.load(f)
+            with codecs.open(self.path, 'r', encoding='utf-8') as f:
```

Comment:
If we're just opening with an encoding, I think the best cross-version code would be `io.open`

I'm basing that opinion on https://stackoverflow.com/a/22288895/6124862 and https://stackoverflow.com/questions/33891373/difference-between-io-open-vs-open-in-python

Reply:
Changed to using `io.open`